public/assets/styles/_sections/Intro.less

```diff
@@ -0,0 +1,116 @@
+@top-spacing: 100px;
+
+#Intro{
+  padding-bottom: 2em;
+  #Logo {
+    display: flex;
+    margin: 2em auto;
+    justify-content: center;
+    font-weight: 100;
+    font-size: 1.5em;
+    letter-spacing: 5px;
+    img{
+      margin-right: 20px;
+    }
+  }
+  .content {
+    width: 90%;
+    margin: 0 auto;
+    text-align: center;
+    h2 {
+      font-size: 33px;
+      font-weight: normal;
+      letter-spacing: -1px;
+      line-height: 50px;
+
+    }
+    p {
+      font-size: 18px;
+      letter-spacing: 0;
+      color: @light-gray;
+      line-height: 30px;
+    }
+
+    .button {
+      margin: 3em 0;
+    }
+  }
+
+  .phones {
+    background-image: url('../images/PLTRFM_TwoPhones.png');
+    background-size: 100%;
+    background-repeat: no-repeat;
+    background-position: center;
+    img {
+      opacity: 0;
+      width: 100%;
+    }
+  }
+
+  .goal {
+    margin-top: 1em;
+    p {
+      line-height: 42px;
+      letter-spacing: -.12px;
+      font-size: 28px;
+    }
+  }
+
+  .logos {
+    text-align: center;
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: center;
+    margin: 0 auto;
+    width: 100%;
+    img {
+      margin: 1em 0.5em;
+    }
+  }
+}
+
+@media only screen and (-webkit-min-device-pixel-ratio: 2), only screen and (-o-min-device-pixel-ratio: 2/1) {
```

Comment:
You can shorten this, and I'd tweak it a bit and use this query for retina (it's more inclusive): `@media (-webkit-min-device-pixel-ratio: 1.5), (min-resolution: 144dpi) {}`